client/boxoptions/options.go

```diff
@@ -19,32 +20,38 @@ var boxOptions = option.Options{
 	DNS: &option.DNSOptions{
 		Servers: []option.DNSServerOptions{
 			{
-				Tag:     "dns-google",
-				Address: "tls://8.8.8.8",
+				Tag:     "dns-google-dot",
+				Address: "tls://8.8.4.4",
+			},
+			{
+				Tag:     "dns-cloudflare-dot",
+				Address: "tls://1.1.1.1",
+			},
+			{
+				Tag:     "dns-sb-dot",
+				Address: "tls://185.222.222.222",
+			},
+			{
+				Tag:             "dns-google-doh",
+				Address:         "https://dns.google/dns-query",
+				AddressResolver: "dns-google-dot",
+			},
+			{
+				Tag:             "dns-cloudflare-doh",
+				Address:         "https://cloudflare-dns.com/dns-query",
+				AddressResolver: "dns-cloudflare-dot",
+			},
+			{
+				Tag:             "dns-sb-doh",
+				Address:         "https://doh.dns.sb/dns-query",
+				AddressResolver: "dns-sb-dot",
```

Comment:
I think we should probably use the same DNS we us here:

https://github.com/getlantern/kindling/blob/main/smart_dialer_config.yml